src/Cassandra.IntegrationTests/Policies/Tests/RetryPolicyTests.cs

```diff
@@ -226,6 +226,40 @@ private void RetryPolicyTest(ITestCluster testCluster)
 
         }
 
+        /// <summary>
+        /// Test IdempotenceAwareRetryPolicy. 
+        /// 
+        /// If write query write is idempotent it will retry according to child retry policy specified.
+        /// 
+        /// @jira_ticket CSHARP-461
+        /// 
+        /// @test_category connection:retry_policy
+        /// </summary>
+        [Test, Category("short")]
+        public void IdempotenceAwareRetryPolicy_ShouldUseChildRetryPolicy_OnWriteTimeout()
+        {
+            const string keyspace = "idempotenceAwarepolicytestks";
+            var testCluster = TestClusterManager.GetNonShareableTestCluster(2);
```

Comment:
`TestClusterManager.GetNonShareableTestCluster()` is deprecated see [xml doc](https://github.com/datastax/csharp-driver/blob/master/src/Cassandra.IntegrationTests/TestClusterManagement/TestClusterManager.cs#L96).

Use `CreateNew()` instead.